src/ansys/geometry/core/plotting/plotter_helper.py

```diff
@@ -229,10 +231,13 @@ def compute_edge_object_map(self) -> Dict[pv.Actor, EdgePlot]:
         """
         for object in self._geom_object_actors_map.values():
             # get edges only from bodies
+            geom_obj = object.object
             if (
-                isinstance(object, Body)
-                or isinstance(object, MasterBody)
-                or isinstance(object, Face)
+                isinstance(geom_obj, Body)
+                or isinstance(geom_obj, MasterBody)
+                or isinstance(geom_obj, Face)
+                or isinstance(geom_obj, SketchFace)
+                or isinstance(geom_obj, Sketch)
```

Comment:
Shouldn't it be SketchEdge?

Reply:
In this part of the code we need to get the objects that have a list of edges as an attribute, not the edges themselves